dockstore-cli-integration-testing/src/test/java/io/dockstore/client/cli/GeneralWorkflowRegressionIT.java

```diff
@@ -83,7 +83,7 @@ public static void getOldDockstoreClient() throws IOException {
         FileUtils.copyURLToFile(url, dockstore);
         assertTrue(dockstore.setExecutable(true));
         url = new URL("https://raw.githubusercontent.com/DockstoreTestUser2/md5sum-checker/1.6.0/checker-input-cwl.json");
-        md5sumJson = temporaryFolder.newFile("md5sum-wrapper-tool.json");
+        md5sumJson = temporaryFolder.newFile("md5sum_cwl.json");
```

Comment:
Is this change intentional? 
It doesn't seem obviously related to singularity testing, but I could be wrong.

Reply:
Yes, I changed it because that json is input for the CWL version of the md5sum checker. I needed to make a second json file to use with the WDL version, so it seemed to make more sense to call them md5sum_cwl.json and md5sum_wdl.json, to show that they are variations of the same thing